src/js/keyboardInput.js

```diff
@@ -226,4 +240,63 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         }
     };
 
+    fluid.defaults("gpii.firstDiscovery.keyboardInputTts", {
+        invokers: {
+            speak: {
+                func: "{fluid.textToSpeech}.queueSpeech"
+            },
+            speakOnFocusMessage: {
+                funcName: "gpii.firstDiscovery.keyboardInputTts.speakOnFocusMessage",
+                args: ["{that}", "{gpii.firstDiscovery.keyboardInput}.container"]
+            },
+            speakShiftOnLatch: {
+                funcName: "gpii.firstDiscovery.keyboardInputTts.speakShiftOnLatch",
+                args: [
+                    "{that}",
+                    "{gpii.firstDiscovery.keyboardInput}.model.shiftLatched",
+                    "{that}.msgLookup.shiftLatched"
+                ]
+            }
+        },
+        modelListeners: {
+            "shiftLatched.speakShiftOnLatch": "{that}.speakShiftOnLatch"
+        },
+        listeners: {
+            // This keypress.speak listener has a priority of "first"
+            // as it needs to happen before the keyboard assessment
+            // check is run. That check will cause the panel to be
+            // re-rendered and we need to get our speech queued before
+            // the re-rendering happens.
+            "keypress.speak": {
+                listener: "{that}.speak",
+                args: ["{arguments}.0"],
+                priority: "first"
```

Comment:
Until we get the FLUID-5506 branch merged in, we should adjust this to some fixed priority such as -10, say, to make it at least possible for the priority to be beaten elsewhere.